docs/user/alerts.md

```diff
@@ -59,3 +59,55 @@ should investigate the logs associated with your cloud provider controllers and
 the Machine API resources to discover the root cause. For more information on
 why nodes, or machines, might not become ready please see the
 [Machine API FAQ](https://github.com/openshift/machine-api-operator/blob/master/FAQ.md).
+
+## ClusterAutoscalerUnableToScaleCPULimitReached
+The number of total cores in the cluster has exceeded the maximum number set on the
+cluster autoscaler. This is calculated by summing the cpu capacity for all nodes
+in the cluster and comparing that number against the maximum cores value set for the
+cluster autoscaler (default 320000 cores).
```

Comment:
Is this default correct? Seems absurdly high? Do we have a source for that value? If so, is it worth adding a link inline?

Reply:
this is just the default if you set nothing on the cluster autoscaler, see [this faq entry](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-are-the-parameters-to-ca), also we do allow [not setting those limits in a ClusterAutoscaler](https://github.com/openshift/cluster-autoscaler-operator/blob/master/pkg/apis/autoscaling/v1/clusterautoscaler_types.go#L77-L94)

Reply:
Could you add a description how you can get the current maxmum settings? like "You can run this to get what's the maximum cores setup in this cluster".

Reply:
@wanghaoran1988 Does this need to happen before we merge or could that follow in a later release? I'm happy with this personally and would prefer to merge it today so its in before feature freeze, but Mike is out on PTO today so wouldn't have time to update this before FF

Reply:
I'm ok with a follow release.